doc/man/8/crushtool.rst

```diff
@@ -108,6 +108,13 @@ pools; it only runs simulations by mapping values in the range
    shows that value **24** is mapped to devices **[11,6]** by rule
    **1**.
 
+   One of the following options is required when using the ``--show-mappings``
```

Comment:
Suggest s/options// and changing `(1)` / `(2)` to `(a)` / `(b)`, then making a/b list items rather than one paragraph

Reply:
I've taken both of these suggestions. They're in the text now.